eahub/templates/eahub/group.html

```diff
@@ -54,5 +54,42 @@ <h1>{{ group.name }}</h1>
       {% include 'eahub/maps/profile_page_map.html' with lat=group.lat|default_if_none:0 lon=group.lon|default_if_none:0 %}
     </div>
   </div>
+
+  <div class="claim_group_container">
+  {% if user.is_authenticated %}
```

Comment:
I changed my mind. Users should see the claim button even if they're not logged in. If somebody organizes a group we want them to be invited to claim it (which involves creating an account if they don't already have one).

Reply:
Made the button visible, but once you click it, it now says "You need to be logged in to claim a group". If they're not logged in, we don't have their email address, so there's no point in letting them claim it